View/SystemInfoViewBuilder.php

```diff
@@ -47,14 +49,9 @@ public function buildView(array $parameters)
      * @param string $identifier A SystemInfo collector identifier (php, hardware...)
      *
      * @return \EzSystems\EzSupportToolsBundle\SystemInfo\Collector\SystemInfoCollector
-     * @throws \eZ\Publish\Core\Base\Exceptions\NotFoundException If no SystemInfoCollector exists with this identifier
      */
     private function getCollector($identifier)
```

Comment:
@bdunogier Do you want to keep this method, given that it doesn't do much anymore?